scripts/o3de/o3de/export_project.py

```diff
@@ -608,6 +609,11 @@ def build_assets(ctx: O3DEScriptExportContext,
         logger.info(f"Processing assets for {ctx.project_name}")
 
     cmake_build_assets_command = [asset_processor_batch_path, "--project-path", ctx.project_path]
+    if selected_platform:
+        if type(selected_platform) == type([]):
+            cmake_build_assets_command.extend([f'--platforms={",".join(selected_platform)}'])
+        else:
+            cmake_build_assets_command.extend([f'--platforms={selected_platform}'])
```

Comment:
I think you can more strict about the argument type for `selected_platform` in this function and force it to be either a list of platforms or None. There is no real value of adding that flexibility especially since its unlikely the method is called from outside of this package. I say this because I see that the `bundle_assets` function below only accepts a list of strings.